tests/cypress-smoke.json

```diff
@@ -17,6 +17,7 @@
   "testFiles": [
     "cluster/test-check-extensions.spec.js",
     "namespace/run-before.spec.js",
+    "namespace/test-service-management.spec.js",
```

Comment:
Rename the file and change here for something like `test-smoke-...` or `test-check-...`